### PR TITLE
Add apps methods

### PR DESCRIPTION
**Pull request by g1y**

### CODE REVIEW

File: lib/api/apps/apps.rb [hunk 1]
```diff
@@ -1,13 +1,37 @@
...
+
 	def list()
+		method = "GET"
+		return self.client.request(method, self.ENDPOINT)
 	end
 
-	def delete()
+	def delete(appid)
+		method = "DELETE"
+		if not appid.is_a? Integer
```
Comment (djmetzle) [inline lib/api/apps/apps.rb]: `if not X === unless X`

Comment (g1y) [reply lib/api/apps/apps.rb]: :+1:

**Comment by djmetzle** [comment]
Conflicts?

**Comment by g1y** [comment]
They gone

**Comment by djmetzle** [comment]
CR :+1:

**Comment by djmetzle** [comment]
CR :+1:

**Pull request merged**